### PR TITLE
fixing teardown (Hopefully)

### PR DESCRIPTION
**Pull request by SK1Y101**

### CODE REVIEW

File: roles/maas_postgres_primary/tasks/teardown.yaml [hunk 1]
```diff
@@ -1,11 +1,5 @@
 ---
 
-- name: "Install psycopg2 For Configuration"
-  ansible.builtin.pip:
-    name: "psycopg2"
-    version: "2.9.3"
-    state: "present"
-
 - name: "Remove Replication User"
   community.postgresql.postgresql_user:
```
Comment (SK1Y101) [inline roles/maas_postgres_primary/tasks/teardown.yaml:4]: Actually, @cpg1111 are these tasks even necessary if we remove postgres in the next step?

Comment (cpg1111) [reply roles/maas_postgres_primary/tasks/teardown.yaml:4]: hmm not really, we could remove these altogether.

Comment (SK1Y101) [reply roles/maas_postgres_primary/tasks/teardown.yaml]: excellent

File: roles/maas_postgres/tasks/teardown.yaml [hunk 2]
```diff
@@ -7,35 +7,25 @@
...
     state: absent
 
-- name: "Uninstall Postgres"
-  ansible.builtin.apt:
-    name: "{{ maas_postgres_deb_name | default('postgresql-{{ maas_postgres_version_number') }}"
-    update_cache: true # update cache still if previous packages are already installed
+- name: "Uninstall psycopg2"
+  ansible.builtin.pip:
+    name: "psycopg2"
+    version: "2.9.3"
```
Comment (cpg1111) [inline roles/maas_postgres/tasks/teardown.yaml:29]: why hardcode the version?

**Review by cpg1111 — APPROVED** [review]
One small nit, but shouldn't hold up merge.

File: roles/maas_postgres/tasks/teardown.yaml (hunk 2 above)
Comment (SK1Y101) [reply roles/maas_postgres/tasks/teardown.yaml:29]: I think the install task that I copied had this hardcoded, but yeah that shouldn't be needed for teardown

**Pull request merged**